apps/analog-app/src/app/pages/search.server.ts

```diff
@@ -0,0 +1,35 @@
+import type { PageServerLoad } from '@analogjs/router';
+import {
+  fail,
+  json,
+  redirect,
+  type PageServerAction,
+} from '@analogjs/router/server/actions';
+import { getQuery, readFormData } from 'h3';
+
+export async function load({ event }: PageServerLoad) {
+  const query = getQuery(event);
+  console.log('loaded search', query['search']);
+
+  return {
+    loaded: true,
+    searchTerm: `${query['search']}`,
+  };
+}
+
+export async function action({ event }: PageServerAction) {
```

Comment:
nit: this is copy-pasta so not needed? If this is for demo purposes, can we change the body to something related to the search page instead?

Reply:
Oh yea, copy-pasta. Will remove